src/jetpack-editor-setup.js

```diff
@@ -44,16 +44,18 @@ export default ( jetpackState ) => {
 
 	const jetpackData = setJetpackData( jetpackState );
 
-	const mediaFilesCollectionBlock = useSelect( ( select ) => {
-		return select( 'core/block-editor' ).getSettings( 'capabilities' )
-			.mediaFilesCollectionBlock;
-	}, [] );
+	// eslint-disable-next-line @wordpress/react-no-unsafe-timeout
```

Comment:
Maybe we could write down the reason why are we using a timeout here too for future references

Reply:
Good idea!  added here 6eb17d5dc